### PR TITLE
January 2021 connectathon updates for running in docker on the remote servers.

### PR DESCRIPTION
**Pull request by plarocque4**

### CODE REVIEW

File: src/util/fhir.js [hunk 1]
```diff
@@ -33,12 +33,12 @@ function getReference(ehrUrl, reference) {
 
 function login() {
 
-    const tokenUrl = this.state.config.auth + "/realms/" + this.state.config.realm + "/protocol/openid-connect/token"
+    const tokenUrl = (process.env.REACT_APP_AUTH ? process.env.REACT_APP_AUTH : this.state.config.auth) + "/realms/" + (process.env.REACT_APP_REALM ? process.env.REACT_APP_REALM : this.state.config.realm) + "/protocol/openid-connect/token"
```
Comment (KeeyanGhoreshi) [inline src/util/fhir.js:36]: as far as I can tell this login function isn't used anywhere and can be deleted.

File: src/components/RequestBox/RequestBox.js [hunk 2]
```diff
@@ -317,11 +317,12 @@ export default class RequestBox extends Component {
     );
 
     client
-      .request("Patient?_sort=identifier&_count=8", { flat: true })
+      .request("Patient?_sort=identifier&_count=12", { flat: true })
       .then((result) => {
         this.setState({
           patientList: result,
         });
+        console.log(result);
```
Comment (KeeyanGhoreshi) [inline src/components/RequestBox/RequestBox.js]: perhaps an errant console log?

**Review by KeeyanGhoreshi — APPROVED** [review]
Looks good, left some comments but neither is a big deal

File: src/components/RequestBox/RequestBox.js (hunk 2 above)
Comment (plarocque4) [reply src/components/RequestBox/RequestBox.js]: Got rid of the log.

File: src/util/fhir.js (hunk 1 above)
Comment (plarocque4) [reply src/util/fhir.js:36]: Let's fix this with a separate PR, I don't want to go through testing that now.

**Pull request merged**